app/src/main/res/navigation/nav_graph.xml

```diff
@@ -979,4 +986,14 @@
                 app:argType="string" />
         </fragment>
     </navigation>
+
+    <navigation
+        android:id="@+id/credit_cards_graph"
```

Comment:
Why is this a separate `navigation` graph?

Reply:
This will have similar number of fragments and action to `search_engine_graph` in the future. I am doing this based on the assumption that this is a good practise (but unverified). Is there any reason we should keep this in the larger navigation graph?